### ISSUE TITLE
Feature request: support for other hashcat modes

### ISSUE THREAD

**Issue by recon-ng** [fireeye/gocrack#17]
I don't believe gocrack currently supports any other mode beyond dictionary-from-file-with-rules, and bruteforce-masks.

Would be nice to see: dictionary-with-multiple-rules, dictionary-from-stdin-with-multiple-rules, markov-bruteforce, combinator, hybrid.

**Comment by tankbusta** [comment]
Yes, you are correct as it was built around how we use hashcat as a company. It may be a bit until I can work on adding additional modes but PR's are welcome :D

**Comment by clem9669** [comment]
+1
